config/config.go

```diff
@@ -921,17 +922,32 @@ func fetchConfig(u *url.URL) ([]byte, error) {
 	}
 	req.Header.Add("Accept", "application/toml")
 	req.Header.Set("User-Agent", internal.ProductToken())
-	resp, err := http.DefaultClient.Do(req)
-	if err != nil {
-		return nil, err
-	}
 
-	if resp.StatusCode != http.StatusOK {
-		return nil, fmt.Errorf("failed to retrieve remote config: %s", resp.Status)
+	retries := 3
```

Comment:
nitpick but I guess this (retries) could be added to the constants at the top too, for consistency

Reply:
Yeah I get you. I only added the interval var up top just so I could access and change it for the test so it wouldn't take 30s to run